### PR TITLE
Move percy keys from encrypted .travis.yml entries to travis config

### PR DESCRIPTION
**Pull request by rsimha-amp**
Something is up with how Travis consumes encrypted keys from .travis.yml. The tests are now using the keys as set in the travis settings page. See https://travis-ci.org/ampproject/amphtml/builds/231611667 for an example of a successful percy run for a push to master.

Issues #8, #8340, #9069

### CODE REVIEW

**Comment by rsimha-amp** [comment]
@erwinmombay: Ping.

**Comment by rsimha-amp** [comment]
@kmh287 for review.

File: build-system/tasks/visual-diff.js [hunk 1]
```diff
@@ -58,7 +58,9 @@ function extractPercyKeys() {
         'Error: PERCY_TOKEN doesn\'t look like a valid Percy API key'));
     process.exit(1);
   }
-  util.log('Percy token: ', util.colors.magenta('<redacted>'));
+  // Note: The secret percy token only gets printed during local use. Travis
```
Comment (kmh287) [inline build-system/tasks/visual-diff.js]: Who will be able to see the token?

File: .travis.yml [hunk 2]
```diff
@@ -39,8 +39,6 @@ branches:
 env:
   global:
     - NPM_CONFIG_PROGRESS="false"
-    - secure: nvLqQdU2CGwvsQ3E1M/6HgBJAfmSzd90bDYQ1vkAkwGj+VOVQauqj7RDJACJS2XysJjzIvzNUOtKFvTSzThye167sKAZnQMmj9U8ATdlJfv5xq88QkkoCuJIOPtVnOWS8gBrDAy9d+Xwi22ZcE5kAAov1gNDmnX6YcPwW8y09vD+Jb1WkxofwTR5U3dP7pxVNRwgXY12g2lmbj+TZy5RHvjs1/ESSVtt/5Wq+Ubv9O6KAVPrh/VdVTYyvgynvwzz5jG3qe7981W8rUNIt+ihzdxJMSaoQkiBs5UM3qi8ibqKI2KHdDmGQg+vc/c5mjCCd9FAz7lF+fJJQ3YThTLmIN3u7Hil+o92OaAt2knwtZ3HRv/jbVQfnU0L9hqHw/kBYE37O4x4/ToJyR3EgbvFXqgiVph0eDUnsmi8PZc5FUiwb8r85thQmfUB+urkOXbnOyHQ9B74T+UjAvlaqLdQ53/9UTzgvLNm2z6RU65G9QYJVMSzj82K2rZ9AyVwjZMp/iuQDoUcUsuMhKDh2genHnxRtQa/TbLQM05oYHRqFa9oT3Q7C7QSSL8mmi/QMfGlTfxhM49CZsbcU2qbMCMCL7ZGWaSiXiEy7XICOzJnOiKFowDK5ihAZaQpRNquMwUSIMnnqrFgfWDbQSUmkOAwSm+2u2+SsXhT+WQfraRfSSI=
```
Comment (kmh287) [inline .travis.yml]: These were moved elsewhere right? Is it okay that these were publicly visible on github?

Comment (cramforce) [inline .travis.yml]: Yes. This is travis' mechanism to safely do that.

Comment (rsimha-amp) [inline .travis.yml]: These are now part of the travis config, and are in a mode where they are not printed within logs. This is exactly how we store our sauce tokens.

File: build-system/tasks/visual-diff.js (hunk 1 above)
Comment (rsimha-amp) [inline build-system/tasks/visual-diff.js]: Only engineers who want to do a local visual diff run, who already have access to the tokens. Those who access the public travis logs will not see the token in the logs.

**Pull request merged**